webknossos-datastore/app/com/scalableminds/webknossos/datastore/datareaders/DatasetArray.scala

```diff
@@ -30,11 +30,35 @@ class DatasetArray(vaultPath: VaultPath,
 
   protected lazy val chunkReader: ChunkReader = new ChunkReader(header)
 
+  // Helper variables to allow reading 2d datasets as 3d datasets with depth 1
```

Comment:
Could you also add a comment in the header definition mentioning this? We might in the future want to use a value from the header class, and should instead look here.